.github/workflows/nos.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: [3.6, 3.7, 3.8, 3.9, "3.10", "3.11"]
```

Comment:
another question, why are they quoted?

Reply:
Because yaml and/or the gh action yaml parser is weird. Without quoting, it will try to use Python 3.1 .

https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python

Reply:
I see, thanks